Cargo.toml

```
@@ -73,9 +74,10 @@ serial_test = { workspace = true }
 tempfile = { workspace = true }
 
 [workspace.dependencies]
-astarte-device-sdk = { git = "https://github.com/astarte-platform/astarte-device-sdk-rust.git", rev = "ca0325e0db334e1755fa1e56b14ae31658a4b2c7" }
-astarte-device-sdk-mock = { git = "https://github.com/astarte-platform/astarte-device-sdk-rust.git", rev = "ca0325e0db334e1755fa1e56b14ae31658a4b2c7" }
-astarte-message-hub-proto = { git = "https://github.com/astarte-platform/astarte-message-hub-proto.git", rev = "87e9b3937cd563576de4bcb801a91c700bf196bb" }
+astarte-device-sdk = "0.10.2"
+astarte-device-sdk-mock = "0.10.2"
+astarte-interfaces = "0.1.1"
+astarte-message-hub-proto = "0.8.1"
```

Bump to [0.8.3](https://github.com/astarte-platform/astarte-message-hub-proto/releases/tag/v0.8.3)
```suggestion
astarte-message-hub-proto = "0.8.3"
```